### ISSUE TITLE
Provide DisabledPage

### ISSUE THREAD

**Issue by aloscha** [StarcounterApps/Images#87]
We should create another mapping for disabled editable page. I need to have something like that:

If no images inside: ![image](https://cloud.githubusercontent.com/assets/5141943/24466673/ff384e52-14b1-11e7-9896-b3078960f332.png)
If image inside: 
There should be just image preview

@warpech WDYT?

**Comment by warpech** [comment]
If you want just an image preview, then there already is a partial for that.

You just need a conditional map to show one or another partial, depending on your need.

**Comment by aloscha** [comment]
Yes get it, but what about situation when I have no image and my control is disabled - if I present patrial there will be 'Illustration should has a content'

So imho I need some partial with disabled mode - EditContentPage
